pkg/chartutil/values_test.go

```diff
@@ -268,7 +325,7 @@ func TestCoalesceTables(t *testing.T) {
 		t.Fatal("Address went away.")
 	}
 
-	if addr["street"].(string) != "234 Spouter Inn Ct." {
+	if addr["street"].(string) != "123 Spouter Inn Ct." {
```

Comment:
For those watching along at home, this is the test that was broken and led to a false positive.